lib/config.js

```diff
@@ -149,6 +149,10 @@ var normalizeConfig = function (config, configFilePath) {
     throw new Error('Invalid configuration: client.args must be an array of strings')
   }
 
+  if ('browsers' in config && Array.isArray(config.browsers) === false) {
```

Comment:
The only drawback here is if `browsers` is provided as an empty string, but I wonder how often that'll be the case.